### ISSUE TITLE
Document remaining commands

### ISSUE THREAD

**Issue by PMunch** [nim-lang/nimsuggest#95]
According to https://nim-lang.org/docs/nimsuggest.html only `def`, `con`, `sug`, and `use` are explained but according to the `nimsuggest` command we have a longer list (`sug|con|def|use|dus|chk|mod|highlight|outline|known`). It would be nice to have some explanation of these as well.

**Comment by timotheecour** [comment]
agreed but please always refer to devel docs (ie https://nim-lang.github.io/Nim/nimsuggest.html) instead of stable when reporting issues (in case issue was fixed in devel; which in this case isnt' the case)
